### PR TITLE
Merge changelog in deployment flow

### PR DESCRIPTION
**Pull request by hatched**
During user testing it was found that the two change log output formats were confusing to users. This branch merges the two into one, sorted by application type and collapsed by default under a "Model changes" heading.

### CODE REVIEW

**Comment by hatched** [comment]
#### To QA

File: .eslintrc [hunk 1]
```diff
@@ -11,7 +11,7 @@ parserOptions:
 rules:
   comma-dangle: ["error", "never"]
   indent: [2, 2, {SwitchCase: 1, VariableDeclarator: 2}]
-  max-len: [2, 80, 2]
+  max-len: [2, 90, 2]
```
Comment (hatched) [inline .eslintrc]: This was a driveby from a discussion about having to find creative ways of wrapping components. Please try and keep your code lengths under 80 and definitely under 90 if possible. But this provides a little wiggle room instead of having to break right at 80.

File: jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js [hunk 2]
```diff
@@ -773,26 +738,25 @@ class DeploymentFlow extends React.Component {
...
-          groupedChanges={this.props.groupedChanges}
-          listPlansForCharm={this.props.listPlansForCharm}
-          parseTermId={this._parseTermId.bind(this)}
-          servicesGetById={this.props.servicesGetById}
-          showChangelogs={this.state.showChangelogs}
-          showTerms={this.props.showTerms}
-          withPlans={this.props.withPlans} />
-      </juju.components.DeploymentSection>);
+      <div className="deployment-services">
+        <juju.components.AccordionSection
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js]: In order to default these to collapsed, convert from the deploymentSection wrapper component to the AccordionSection component.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7895/

File: jujugui/static/gui/src/app/components/budget-table/row/row.js [hunk 3]
```diff
@@ -409,7 +393,9 @@ class BudgetTableRow extends React.Component {
           <div>
             {this._generateSharedFields()}
             {this._generatePlanCols()}
-            {this._generateExtra()}
+            <div className="twelve-col no-margin-bottom">
```
Comment (Lukewh) [inline jujugui/static/gui/src/app/components/budget-table/row/row.js]: Should this not be in a separate `this._generateExtras()` or at least in an if so we don't have empty layout divs?

File: jujugui/static/gui/src/app/components/budget-table/row/test-row.js [hunk 4]
```diff
@@ -133,7 +133,7 @@ describe('BudgetTableRow', function() {
               </div>
               {undefined}
             </div>
-            {undefined}
+            <div className="twelve-col no-margin-bottom" />
```
Comment (Lukewh) [inline jujugui/static/gui/src/app/components/budget-table/row/test-row.js]: Adding a conditional would remove this empty div.. And all the ones int he tests that follow.

File: jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js (hunk 2 above)
Comment (Lukewh) [inline jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js]: It's a good section bront

File: jujugui/static/gui/src/app/components/deployment-flow/machines/machines.js [hunk 5]
```diff
@@ -81,7 +81,7 @@ class DeploymentMachines extends React.Component {
     let chargeMessage = '';
     const cloudName = this.props.cloud ? this.props.cloud.name : 'the cloud';
     if (cloudName !== 'localhost') {
-      chargeMessage = 'You will incur a charge from your cloud provider.';
+      chargeMessage = 'You may incur charges from your cloud provider.';
```
Comment (Lukewh) [inline jujugui/static/gui/src/app/components/deployment-flow/machines/machines.js]: nice

File: jujugui/static/gui/src/app/components/deployment-flow/services/_services.scss [hunk 6]
```diff
@@ -1,4 +1,17 @@
...
+  width: 1030px;
+  background-color: $light-grey;
+  padding: 20px;
+
+  .accordion-section {
+    border: none;
+  }
+
+  .accordion-section__title {
+    font-size: 23px;
```
Comment (Lukewh) [inline jujugui/static/gui/src/app/components/deployment-flow/services/_services.scss]: `1.4375rem`s

File: jujugui/static/gui/src/app/components/budget-table/row/row.js (hunk 3 above)
Comment (hatched) [inline jujugui/static/gui/src/app/components/budget-table/row/row.js]: There are no layout divs in practice, only in the tests because populating the extra info is difficult :)

File: jujugui/static/gui/src/app/components/deployment-flow/services/_services.scss (hunk 6 above)
Comment (hatched) [inline jujugui/static/gui/src/app/components/deployment-flow/services/_services.scss]: updated

File: jujugui/static/gui/src/app/components/budget-table/row/test-row.js (hunk 4 above)
Comment (hatched) [inline jujugui/static/gui/src/app/components/budget-table/row/test-row.js]: see above

**Comment by Lukewh** [comment]
Code looks good @hatched 👍

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7897/

**Comment by hatched** [comment]
Need to land to avoid conflicts

:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge
